### DISCUSSION TITLE
Please provide me with the download link for the apm-trace-ignore-plugin-9.3.0. jar package. I have searched many places but couldn't find it. Thank you all~

### DISCUSSION THREAD

**Discussion by huang-sir-001** [apache/skywalking#12662] [Q&A] [answered]
Please provide me with the download link for the apm-trace-ignore-plugin-9.3.0. jar package. I have searched many places but couldn't find it. Thank you all~

**Comment by wu-sheng** [comment]
Why don't you download from the website? The official release included that.

**Reply by huang-sir-001** [reply]
I downloaded it, but I couldn't find this jar package in the/skywalking agent/plugins directory
My version is apache-skywalking-java-agent-9.3.0.tgz

**Reply by huang-sir-001** [reply]
I'm very sorry to bother you with this matter, but I searched for a long time but couldn't find it before choosing to seek help

**Reply by wu-sheng** [reply, marked as answer]
Because you don't read documents, but relying on some unclear blogs.

This is an optional plugin, it is in optional folder, and you need to activate it manually.

Again, read documents. Blog things, we can't clarify what they said.

**Reply by huang-sir-001** [reply]
Thank you, I will go over the document again

**Reply by huang-sir-001** [reply]
Thank you very much. I have found the directory of optional plugins. I apologize for disturbing you and thank you for your help
